translate/automl/translate_v3_batch_translate_text_with_model.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START translate_v3_batch_translate_text_with_model]
+from google.cloud import translate
+
+
+def batch_translate_text_with_model(
+    input_uri="YOUR_INPUT_URI",
+    output_uri="YOUR_OUTPUT_URI",
```

Comment:
Does these have to be gcs uris?
What type of input file? 
Is the output uri a directory?

```
input_uri="gs://YOUR_BUCKET_ID/path/to/your/file.type",
output_uri="gs://YOUR_BUCKET_ID/path/to/save/results/"
```